plugins/com.google.cloud.tools.eclipse.util/src/com/google/cloud/tools/eclipse/util/MavenUtils.java

```diff
@@ -42,6 +49,18 @@
 
 public class MavenUtils {
 
+  /**
+   * An extension to {@link java.util.concurrent.Callable} that may throw an exception.
```

Comment:
`Callable` can already thrown an `Exception`, but I guess you want to make it specific? This comment can be misleading anyway.